recipes-bsp/libimxdmabuffer/libimxdmabuffer_1.1.2.bb

```diff
@@ -18,13 +18,17 @@ S = "${WORKDIR}/git"
 
 inherit pkgconfig waf use-imx-headers ptest
 
-# Device node path to the DMA-BUF heap to use in the dma-heap allocator.
-# This must be one that allocates physically contiguous memory, otherwise
-# it cannot allocate usable DMA buffers.
-DMA_HEAP_DEVICE_NODE_PATH = "/dev/dma_heap/linux,cma"
+# Device node path to the DMA-BUF heap to use in the dma-heap allocator. This
+# must be one that allocates physically contiguous memory, otherwise it cannot
+# allocate usable DMA buffers. An uncached DMA heap is selected to avoid issues
+# with cache coherence. Note that the uncached heap is not available prior to
+# imx-kernel 5.15.5; older kernels should use "/dev/dma_heap/linux,cma" instead
+# and remove the "--dma-heap-uncached-memory" switch below from EXTRA_OECONF.
+DMA_HEAP_DEVICE_NODE_PATH = "/dev/dma_heap/linux,cma-uncached"
```

Comment:
The node path ought to be handled at compile time or runtime. If possible to do it at runtime, it'd be compatible to new and old kernels.

If compile time is the only option here tie it to a PACKAGECONFIG so it is always used right.